setup.py

```diff
@@ -42,7 +42,7 @@ def read(fname):
     install_requires=[], # use pip -r requirements.txt instead
     entry_points={
         'console_scripts': [
-                'socorro = socorro.app.socorro_app:SocorroWelcomeApp.run'
+                'socorro = socorrolib.app.socorro_app:SocorroWelcomeApp.run'
```

Comment:
Apps have always been confusing to me. But it's kinda strange to put apps in a package called "somethinglib". An app is usually the last thing to write when you have libraries and dependencies sorted out. 
Perhaps it's hair-splittingly distracting to obsess about what should be moved and what should stay. But it's almost always a good idea to stop and think.